benchmark/benchmark/run_benchmark.py

```diff
@@ -63,25 +69,25 @@ def prepare_questions(kwargs):
     return questions, url_to_content
 
 def parse_response(response, test_q):
-        result = json.loads(response[0])
-        test_q["p_yes"] = float(result["p_yes"])
-        test_q["p_no"] = float(result["p_no"]) 
-        if response[2] is not None:
-            test_q["input_tokens"] = response[2].cost_dict["input_tokens"]
-            test_q["output_tokens"] = response[2].cost_dict["output_tokens"]
-            test_q["total_tokens"] = response[2].cost_dict["total_tokens"]
-            test_q["input_cost"] = response[2].cost_dict["input_cost"]
-            test_q["output_cost"] = response[2].cost_dict["output_cost"]
-            test_q["total_cost"] = response[2].cost_dict["total_cost"]
-            test_q["prompt_response"] = response[1].replace(os.linesep, "")
-        if float(result["p_yes"]) == float(result["p_no"]):
-            test_q["prediction"] = None
-        else:
-            test_q["prediction"] = (
-                "yes" if test_q["p_yes"] > test_q["p_no"] else "no"
-            )
-        test_q["Correct"] = test_q["prediction"] == test_q["answer"]
-        return test_q
+    result = json.loads(response[0])
+    test_q["p_yes"] = float(result["p_yes"])
+    test_q["p_no"] = float(result["p_no"]) 
+    if response[3] is not None:
+        test_q["input_tokens"] = response[3].cost_dict["input_tokens"]
+        test_q["output_tokens"] = response[3].cost_dict["output_tokens"]
+        test_q["total_tokens"] = response[3].cost_dict["total_tokens"]
+        test_q["input_cost"] = response[3].cost_dict["input_cost"]
+        test_q["output_cost"] = response[3].cost_dict["output_cost"]
+        test_q["total_cost"] = response[3].cost_dict["total_cost"]
+    test_q["prompt_response"] = response[1].replace(os.linesep, "")
+    if float(result["p_yes"]) == float(result["p_no"]):
```

Comment:
Would like to discuss about this validation of the probability outside the PR :)